redisgears_v8_plugin/src/v8_backend.rs

```diff
@@ -286,23 +285,17 @@ impl BackendCtxInterfaceInitialised for V8Backend {
                         }
                     });
 
-                let api_version_supported = initialize_globals_for_version(
-                    api_version,
-                    &script_ctx,
-                    &globals,
-                    &isolate_scope,
-                    &ctx_scope,
-                    config,
-                )?;
+                let api_version_supported: ApiVersionSupported = api_version.try_into()?;
 
-                if let Err(errors) = api_version_supported.validate_code(code) {
-                    let messages = errors
-                        .iter()
-                        .enumerate()
-                        .map(|(index, error)| format!("\t{index}. {}", error.get_msg()))
-                        .collect::<Vec<String>>()
-                        .join("\n");
+                let messages = api_version_supported
+                    .validate_code(code)
+                    .iter()
+                    .enumerate()
+                    .map(|(index, error)| format!("\t{}. {}", index + 1, error.get_msg()))
+                    .collect::<Vec<String>>()
+                    .join("\n");
```

Comment:
I do not think its a good idea to log with `\n` it will work but will not look so good on the Redis log file. Maybe we can log each entry separately?